### DISCUSSION TITLE
Upgrade BERTopic in Visual Studio

### DISCUSSION THREAD

**Discussion by Nee-DS** [MaartenGr/BERTopic#989] [Q&A] [answered]
Not able to update BERTOpic ==0.13.0,  from 0.9.4, in Visual Studio. 
![Capture](https://user-images.githubusercontent.com/62400278/217169037-f02e6c54-2337-4801-b7a5-c8fcd3b5d49f.PNG)

**Comment by MaartenGr** [comment, marked as answer]
I believe this is similar to the following [issue](https://github.com/MaartenGr/BERTopic/issues/62). As mentioned in your error message, you will need to install Microsoft Visual C++ 14.0 or greater for HDBSCAN to work. You can follow the link in your error message or read through the issue mentioned above.

**Reply by Nee-DS** [reply]
Thank you! I got it resolved. Visual Studio was installed with an error and removing that error made me upgrade the version =13
